packages/mobx-react/package.json

```diff
@@ -36,10 +36,10 @@
     },
     "homepage": "https://mobx.js.org",
     "dependencies": {
-        "mobx-react-lite": "^3.4.0"
+        "mobx-react-lite": "^3.4.3"
     },
     "peerDependencies": {
-        "mobx": "^6.1.0",
+        "mobx": "^6.9.0",
```

Comment:
A guess - I don't know the actual version it's gonna be released with. Dunno what's the correct way to handle it.